SimMuon/RPCDigitizer/python/rpcChamberMasker_cff.py

```diff
@@ -69,9 +69,9 @@ def appendRPCChamberMaskerAtUnpacking(process):
                 muonRPCDigis = cms.PSet(initialSeed = cms.untracked.uint32(789342))
                 )
 
-        process.filteredRPCDigiSequence = cms.Sequence(process.preRPCDigis \
-                                                       + process.muonRPCDigis)
-        process.RawToDigi.replace(process.muonRPCDigis, \
-                                  process.filteredRPCDigiSequence)
+        process.filteredRPCDigiTask = cms.task(process.preRPCDigis,
```

Comment:
uppercase Task?

Reply:
Thanks Kevin (and actually all this can be replaced with `RawToDigi.add(process.preRPCDigis)`, which I did in #25162).